.editorconfig

```diff
@@ -55,6 +55,11 @@ csharp_style_namespace_declarations = file_scoped
 # Brace settings
 csharp_prefer_braces = true # Prefer curly braces even for one line of code
 
+# Whitespace options
+csharp_style_allow_embedded_statements_on_same_line_experimental = false
+csharp_style_allow_blank_lines_between_consecutive_braces_experimental = false
+csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimental = false
```

Comment:
I don't know if @dotnet/aspdoi has discussed these settings but this particular one is actively annoying and I'd much rather we didn't take it.